addons/account/views/report_payment_receipt_templates.xml

```diff
@@ -64,11 +64,11 @@
                                 </tr>
                                 <!-- PAYMENTS/REVERSALS -->
                                 <tr t-foreach="inv._get_reconciled_invoices_partials()[0]" t-as="par">
-                                    <t t-set="payment" t-value="par[2].move_id"/>
-                                    <td><span t-field="payment.date"/></td>
-                                    <td><span t-field="payment.name"/></td>
-                                    <td><span t-field="payment.ref"/></td>
-                                    <t t-set="amountPayment" t-value="-payment.amount_total"/>
+                                    <t t-set="payment" t-value="par[2]"/>
+                                    <td><span t-field="payment.move_id.date"/></td>
+                                    <td><span t-field="payment.move_id.name"/></td>
+                                    <td><span t-field="payment.move_id.ref"/></td>
+                                    <t t-set="amountPayment" t-value="-par[0].credit_amount_currency if inv.is_sale_document() else -par[0].debit_amount_currency"/>
```

Comment:
I'd expect something like `par[0].amount * inv.direction_sign` to be more correct
what do you think @smetl ?

Also your code is incorrect if you follow the same steps for a in_refund 

Given this RBILL with a partial payment of 50 euros
![image](https://github.com/user-attachments/assets/10ef24ca-d3f8-44b2-80c5-9c6baccae954)

And we reconcile it with an entry line of 10 dollars
![image](https://github.com/user-attachments/assets/8fc114d0-807a-48b1-9513-16d2aab24eda)

You get
![image](https://github.com/user-attachments/assets/85693a93-a11c-4f73-86b2-c2f225fbeae1)

Reply:
it works well, except for the sign, `-par[0].amount` is ok